client/state/rewind/retention/types.ts

```diff
@@ -0,0 +1 @@
+export type RetentionPeriod = 7 | 30 | 120 | 365;
```

Comment:
This is a good place for the rest of the types we introduce inline in this state subtree, especially when we repeat them in more than 1 place.